### DISCUSSION TITLE
Click and Jog stopped working

### DISCUSSION THREAD

**Discussion by jomo487** [arkypita/LaserGRBL#1758] [Q&A]
Hello
My click and jog function stopped working.  
heres what I do;

load svg and process

Home -moves to the right and gantry moves back to X-10 Y-10 (after it retracts)

I use the jog controls to bring the laser to front left lets say  X-200 Y-200  
-command monitor shows $J=G91X-200Y-200F2000

Press Set Zero point (G92) button -command monitor shows G92 X0Y0 Z0

Frame,corner,center button works good and engraving is correct.

But, Click and  Jog function sends laser back to X0 Y0 instead of selected spot.
-command monitor shows $J=G90 X210 Y210 F2000 

Version 4.4.1 and 4.7.1 same problem
Any Thoughts?  Thanks

**Comment by arkypita** [comment]
Video of LaserGRBL screen showing this issue can help.
You can use something like bandicam free to record desktop https://www.bandicam.com/free-screen-recorder/ and upload somewhere

**Reply by jomo487** [reply]
https://youtu.be/fA8MnUlk7XE

Please notice that after a successful frame button that I put the cursor on the left square but then the cursor moves towards home. 
Please see my original post above to see the process I use.

thanks arkypita
